src/OpenConext/UserLifecycle/Domain/ValueObject/CollabPersonId.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * Copyright 2018 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace OpenConext\UserLifecycle\Domain\ValueObject;
+
+use Webmozart\Assert\Assert;
```

Comment:
Curious how you're going to resolve this when using deptrac :)

Reply:
Yeah, deptrack will not like this at all :) Will fix this when the time comes